src/convertToObject.js

```diff
@@ -11,8 +11,24 @@
  *
  * @return {object}
  */
+/* eslint-enable */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  sourceString
+    .split(';')
+    .forEach(style => {
```

Comment:
You can use `reduce` method to generate an object instead of creating an empty object and `forEach`

Reply:
Not fixed 😢